tested/dsl/translate_parser.py

```diff
@@ -506,11 +545,16 @@ def _convert_testcase(testcase: YamlDict, context: DslContext) -> Testcase:
     line_comment = ""
     _validate_testcase_combinations(testcase)
     if (expr_stmt := testcase.get("statement", testcase.get("expression"))) is not None:
-        if isinstance(expr_stmt, dict) or context.language != "tested":
+        if (
+            isinstance(expr_stmt, dict | ProgrammingLanguageMap)
+            or context.language != "tested"
+        ):
             if isinstance(expr_stmt, str):
                 the_dict = {context.language: expr_stmt}
             else:
                 assert isinstance(expr_stmt, dict)
```

Comment:
Won't this crash when it is a `ProgrammingLanguageMap`, which is also valid

Or, if `ProgrammingLanguageMap` is a `dict`, why does it need to be added to the previous `isinstance` check?